changelog/unreleased/enhancement-add-action-drop-down-to-app-top-bar

```diff
@@ -0,0 +1,7 @@
+Enhancement: add action drop down top app top bar
```

Comment:
nobody noticed the typo 😓 😅

Reply:
Oh no :( gonna fix this asap as possible !

Reply:
Beste!